regexes/device/mobiles.yml

```diff
@@ -5100,19 +5139,19 @@ Samsung:
     - regex: '(?:SAMSUNG-)?GT-N7105'
       model: 'GALAXY Note II LTE'
       device: 'phablet'
-    - regex: '(?:SAMSUNG-)?SM-G7105'
+    - regex: '(?:SAMSUNG-)?SM-G710[25]'
       model: 'GALAXY Grand 2'
       device: 'phablet'
     - regex: '(?:SAMSUNG-)?SM-G53(?:0F|0FZ|0Y|0H|0FZ|1F|1H)'
       model: 'GALAXY Grand Prime'
       device: 'phablet'
     - regex: '(?:SAMSUNG-)?SM-G532F'
-      model: 'GALAXY Grand Prime Plus'
+      model: 'GALAXY J2 Prime/Grand Prime Plus'
```

Comment:
We should decide to use only one name here. Maybe the more familiar / common name if possible.

Reply:
I don't disagree, but I could not decide as both "Grand Prime" and "J2" exist as other variants, and I don't know if one of these two is more popular.